.travis.yml

```diff
@@ -127,6 +127,15 @@ install:
         travis_retry sudo apt-get -qq -y install libegl1-mesa;
       fi;
 
+    # Install IPython
+    - if [ "${DEPS}" == "full" ]; then
+        if [ "${PYTHON}" == "2.7" ]; then
+          pip install -q ipython[notebook];
+        else
+          conda install --yes --quiet ipython$IPYTHON ipython-notebook;
```

Comment:
you *might* also need the `$IPYTHON` on the `ipython-notebook` call, might as well add it to be safe. Also, make sure this is the last call to conda, otherwise it's possible some other package we install with `conda` later will recommend a higher version, and it will override our previous manual installation of this version. (Please add a note in Travis about this, too.)